docs/user_manual.md

```diff
@@ -189,6 +189,12 @@ docker.resizeTty("containerID", height, width);
 docker.startContainer("containerID");
 ```
 
+### Stop a container
+
+```java
+docker.stopContainer("", 10); // kill after 10 seconds
```

Comment:
That's exactly what I meant to do but I was focusing on the parameter I wasn't sure of first and must've gotten. Sorry, my bad, will fix.